app/models/Country.scala

```diff
@@ -332,3 +333,8 @@ object CountryWithValidationDetails {
   private val sloveniaVatNumberRegex = """^[0-9]{8}$"""
   private val slovakiaVatNumberRegex = """^[0-9]{10}$"""
 }
+
+object GetCountryFromCode {
+
+  def getCountryName(countryCode: String): String = euCountries.filter(_.code == countryCode).map(_.name).head
```

Comment:
I don't think we need an extra object for this function, can we put this within the Country object?

Reply:
getCountryName method moved into Country object